DEVICES.md

```diff
@@ -369,7 +369,7 @@ of device.
 - Moes motion sensor lights (XZ-CGV3)
 - Moes star projector
 - MoesGo dimmer switch
-- Space Dog Music Lamp
+- Space Dog Music Lamp (works for Aurora Smart Galaxy Star from tuya)

```

Comment:
Leave out the "from tuya", it is redundant in an integration that is specifically for tuya devices.